### ISSUE TITLE
Problem Installing the setup

### ISSUE THREAD

**Issue by Tikam02** [rtfd/readthedocs.org#3600]
1)when installing through requirements.txt all modules are not installed.
2) problems in modules

![screenshot from 2018-02-13 00-44-14](https://user-images.githubusercontent.com/10548085/36114806-19b19c24-1057-11e8-9bb6-b8839efc59ad.png)

**Comment by Tikam02** [comment]
Official Pagenation Download page.

https://pythonhosted.org/linaro-django-pagination/installation.html

Installation
Prerequisites

This package requires django 1.2. It is not tested on earlier versions and may not work properly there.

To build the documentation from source you will need sphinx.
Installation Options

There are several installation options available:
Using Ubuntu PPAs

For Ubuntu 10.04 onward there is a stable PPA (personal package archive):

    ppa:linaro-validation/ppa

To add a ppa to an Ubuntu system use the add-apt-repository command:

sudo add-apt-repository ppa:linaro-validation/ppa

After you add the PPA you need to update your package cache:

sudo apt-get update

Finally you can install the package, it is called python-versiontools:

sudo apt-get install python-linaro-django-pagination

Using Python Package Index

This package is being actively maintained and published in the Python Package Index. You can install it if you have pip tool using just one line:

pip install linaro-django-pagination

Using source tarball

To install from source you must first obtain a source tarball from either pypi or from Launchpad. To install the package unpack the tarball and run:

python setup.py install

You can pass --user if you prefer to do a local (non system-wide) installation.

Note

To install from source you will need distutils (replacement of setuptools) They are typically installed on any Linux system with python but on Windows you may need to install that separately.

**Comment by Tikam02** [comment]
it's not running even after installing all pagenation modules.

**Comment by 142ayushkumar** [comment]
@Tikam02 did you followed each and every step? I set up my system today itself and it is working fine.
Also provide the console error which you are getting.

**Comment by stsewd** [comment]
That's because RTD use a custom package from GitHub https://github.com/rtfd/readthedocs.org/blob/master/requirements/pip.txt#L73

Please follow the instructions here https://docs.readthedocs.io/en/latest/install.html

**Comment by Tikam02** [comment]
default environment  is python3 or python2

**Comment by 142ayushkumar** [comment]
mine is Python 2.7.12

**Comment by stsewd** [comment]
@Tikam02 do you mean the python interpreter on which rtd run? Python 2 and 3 (no well tested) are supported. If is about the build environment for documentation, python2 is the default.

**Comment by Tikam02** [comment]
yup i think now it'll work for 2.7 and 3.4 specifically.Please upgrade this platform to 3.6 and if possible we can also make it python package pip rtd  in future development.
Thanks for the information.

**Comment by Tikam02** [comment]
still there is some problem with celery module in 2.7, followed each steps.bug even after istalling celery module.


![screenshot from 2018-02-13 01-47-32](https://user-images.githubusercontent.com/10548085/36117523-e89e5ad8-105f-11e8-8c54-e381dee1e0a6.png)

**Comment by 142ayushkumar** [comment]
@Tikam02  you working on python 2?

**Comment by Tikam02** [comment]
yeah, because i had 2.7, 3.5 and 3.6 environment that's why.

**Comment by Jigar3** [comment]
Your celery module is being installed in python 3.6 and you are running python 2.7.
This is causing ImportError.
Use python --version to check which version of python you are using

**Comment by YatriP** [comment]
I am also facing the same problems

**Comment by stsewd** [comment]
@Tikam02 as @Jigar3 mention, probably is a problem with your virtualenv, try using pip as `python -m pip` or `python3 -m pip` (in your case looks like your virtualenv is python3, you can try running `python3 manage.py migrate`).

**Comment by Tikam02** [comment]
@Jigar3 after running in virtualenv 2.7 and installing celery in 2.7 still there is error.
ImportError: No module named celery

**Comment by Jigar3** [comment]
Upload screenshot

**Comment by Tikam02** [comment]
virtualenv - python2.7
all modules installed manually
error after python2 manage.py migrate
![screenshot from 2018-02-13 15-56-15](https://user-images.githubusercontent.com/10548085/36145203-a8d4b074-10d6-11e8-8313-466054118832.png)

**Comment by Tikam02** [comment]
django.contrib.gis.geos.error.GEOSException: Could not parse version info string "3.6.2-CAPI-1.10.2 4d2925d6"
